docker/docker-compose.yml

```diff
@@ -33,10 +33,13 @@ services:
 
   nodejs:
     image: fidals/se-nodejs:dev
-    volumes_from:
-      - app
+    # For refarm-site's front development
```

Comment:
Why did you remove `volumes_from`? How will we perform `gulp watch/build` for local changes?